### PR TITLE
second commit

### PR DESCRIPTION
**Pull request by GeoDevH**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,13 +8,25 @@ <h1>HomeWork</h1>
...
             <li>Firt item</li>
             <li>Second item</li>
             <li>Third item</li>
             <li>Fouth item</li>
             <li>Fifth item</li>
         </ol>
+        <center><h1>HomeWork Completed</h1></center>
+        <center><h2>&#x1F44D;</h2></center>
+        <p>Second table will be added below</p>
+            <h2>&#x1F447;</h2>
```
Comment (RomanIlkiv) [inline index.html]: unnecessary space (пробіл)

File: index.html [hunk 2]
```diff
@@ -8,13 +8,25 @@ <h1>HomeWork</h1>
...
+        <ul>
+            <li>First item</li>
+            <li>Second item</li>
+            <li>Third item</li>
+            <li>Fouth item</li>
+            <li>Fifth item</li>
+        </ul>
+            <h2>&#x1F525; &#x1F525; &#x1F525; &#x1F525; &#x1F525;</h2>
```
Comment (RomanIlkiv) [inline index.html]: unnecessary space (пробіл)

ul and h2 are on the same level here

File: index.html [hunk 3]
```diff
@@ -8,13 +8,25 @@ <h1>HomeWork</h1>
...
-        <h2>&#x1F447;</h2>
+            <h2>&#x1F447;</h2>
         <ol>
             <li>Firt item</li>
             <li>Second item</li>
             <li>Third item</li>
             <li>Fouth item</li>
             <li>Fifth item</li>
         </ol>
+        <center><h1>HomeWork Completed</h1></center>
```
Comment (RomanIlkiv) [inline index.html]: should in another order, like this:
`<h1><center>HomeWork Completed</center></h1>`

File: index.html [hunk 4]
```diff
@@ -8,13 +8,25 @@ <h1>HomeWork</h1>
...
+        <center><h2>&#x1F44D;</h2></center>
```
Comment (RomanIlkiv) [inline index.html]: the same here

File: index.html (hunk 1 above)
Comment (GeoDevH) [reply index.html]: ok

File: index.html (hunk 2 above)
Comment (GeoDevH) [reply index.html]: +

Comment (GeoDevH) [reply index.html]: here "plus"

File: index.html (hunk 3 above)
Comment (GeoDevH) [reply index.html]: accepted

File: index.html (hunk 4 above)
Comment (GeoDevH) [reply index.html]: accepted too

**Review by RomanIlkiv — APPROVED** [review]

**Pull request merged**